travis/prepare/prepare-mongodb

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+
+if [ -f $SYLIUS_CACHE_DIR/composer-mongodb.lock ]; then
+    cp $SYLIUS_CACHE_DIR/composer-mongodb.lock composer.lock
+else
+    composer require doctrine/mongodb-odm="1.0.*@dev" --no-update
```

Comment:
Just copied it from original Travis configuration file, feel free to contribute :)